src/components/Swap.vue

```diff
@@ -218,14 +224,28 @@ export default class Swap extends Mixins(TranslationMixin, LoadingMixin) {
    * Update token From balance and amount (after swap it should be recalculated)
    */
   get tokenFromBalance (): string {
-    return this.tokenFrom?.balance ?? ''
+    if (!this.tokenFrom?.balance) {
+      return ''
+    }
+    const updatedAsset = this.accountAssets.find(asset => asset.address === this.tokenFrom.address)
```

Comment:
I think this actions shouldn't be called inside `tokenFromBalance`, `tokenToBalance `, because `handleSwitchTokens ` method already used for switching (we could remove comments above this computed property declaration)

Reply:
We with @Nikita-Polyakov have decided to keep it for now and refactor later.